index.html

```diff
@@ -0,0 +1,333 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+    </header>
+    <main>
+      <div class="intro">
+        <div class="intro-icon">
+          <img src="./media/svg/icon.svg" alt="Icon for the side" />
+        </div>
+        <h2 class="intro-greet">
+          Hey There.
+          <br />
+          I'm Aziz
+        </h2>
+        <h2 class="intro-me">I’am a Software Developer</h2>
+        <p class="intro-text">
+          I can help you build a product, feature or website <br />
+          Look through some of my work and experience! <br />
+          If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="social">
+          <a
+            class="social-link"
+            href="https://github.com/SpaYco"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
```

Comment:
- [ ] The social icons should be aligned to the left of the intro text. Kindly review the Figma design 

![contact form misaligned icons with arrow](https://user-images.githubusercontent.com/87413774/163387234-c68257e6-2527-48fa-9c06-73ef5585267e.png)